### ISSUE TITLE
Plt not displayed anymore

### ISSUE THREAD

**Issue by OldIMP** [millejoh/emacs-ipython-notebook#730]
## Problem description

After `plt.show()`, no figure was displayed but only `Saving figure <plt_name>`

## Steps to reproduce the problem

`plt.show()`

## System info:
```cl
("EIN system info"
 :emacs-version "GNU Emacs 26.3 (build 1, x86_64-w64-mingw32)
 of 2019-08-29"
 :emacs-bzr-version "96dd0196c28bc36779584e47fffcca433c9309cd"
 :window-system w32
 :emacs-variant nil
 :os
 (:uname nil
	 :lsb-release nil)
 :notebook
 "Name: notebook
Version: 6.0.3
Summary: A web-based notebook environment for interactive computing
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: c:\\users\\shouh\\anaconda3\\envs\\tf1\\lib\\site-packages
Requires: prometheus-client, ipykernel, nbconvert, jupyter-core, jupyter-client, terminado, pyzmq, jinja2, traitlets, tornado, ipython-genutils, nbformat, Send2Trash
Required-by: widgetsnbextension, nbdime, jupyterlab, jupyterlab-server
"
 :ipython "7.10.1
"
 :image-types
 (svg png gif tiff jpeg xpm xbm pbm)
 :image-types-available
 (svg png gif tiff jpeg xpm xbm pbm)
 :request
 (:backend curl)
 :ein
 (:version "20200712.24"
	   :source-dir "c:/Users/shouh/AppData/Roaming/.emacs.d/elpa/ein-20200712.24/")
 :lib
 ((:name "websocket"
	 :path "~/.emacs.d/elpa/websocket-20200419.2124/websocket.elc"
	 :featurep t
	 :version-var websocket-version
	 :version "1.12")
  (:name "anaphora"
	 :path "~/.emacs.d/elpa/anaphora-20180618.2200/anaphora.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "request"
	 :path "~/.emacs.d/elpa/request-20200517.1305/request.elc"
	 :featurep t
	 :version-var request-version
	 :version "0.3.0")
  (:name "deferred"
	 :path "~/.emacs.d/elpa/deferred-20170901.1330/deferred.elc"
	 :featurep t
	 :version-var deferred:version
	 :version "0.5.0")
  (:name "polymode"
	 :path "~/.emacs.d/elpa/polymode-20200606.1106/polymode.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "exec-path-from-shell"
	 :path "~/.emacs.d/elpa/exec-path-from-shell-20200526.324/exec-path-from-shell.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "dash"
	 :path "~/.emacs.d/elpa/dash-20200524.1947/dash.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "with-editor"
	 :path "~/.emacs.d/elpa/with-editor-20200617.1234/with-editor.elc"
	 :featurep t
	 :version-var nil
	 :version nil)))
```

**Comment by dickmao** [comment]
matplotlib.is_interactive()  should return True.  If so, then plt.show() is not applicable.  Rather, plt.plot() is the rendering function.  I would google this distinction.

**Comment by OldIMP** [comment]
I've tried `plot()` but got only an empty array as output:

```
Saving figure some_digit_plot

Out [16]:
[]
```

`show()` works in the browser

**Comment by dickmao** [comment]
```
emacs -Q -f package-initialize --eval "(setq ein:output-area-inlined-images t)"
```
`plt.plot()` does the rendering and has a return value.
`plt.show()` does nothing without plt.plot() and has no return value.
I would google this distinction.

![oldimp](https://user-images.githubusercontent.com/7578770/88062528-f2c6aa80-cb36-11ea-9526-f40a320e1179.gif)

**Comment by OldIMP** [comment]
Sry for the poor description. I think I found some clue in *Messages*:
> ein: [warn] ein:cell-append-mime-type: no viewer method found in mailcap

**Comment by OldIMP** [comment]
I did some debug logging in `ein:cell-append-mime-type`:

> 21:34:54:906: [verbose] image: (image :type png :data \211PNG
> 
> 
> \0\0\0
> IHDR\0\0\210\0\0\0\373\0\0\0p\202\375\0\0\0sBIT|d\210\0\0\0	pHYs\0\0\0\0\322\335~\374\0\0\08tEXtSoftware\0matplotlib version3.1.2, http://matplotlib.org/%\213\241J\0\0 \0IDATx\234\355\335w|\225\365\371\377\361\327\225\275\201f{$aH@T\\265B\353B\304U*\253V\253h\245*Z\261\326\352\267Z\304b\340\300QAQ\367-F\200 \204=d$a\204\262\257\337I\372\213\361\204\220p\222;\347\234\353\371x\234\207\311\275\362>\347\235{}nQU\2141\306\230\232\374\234`\2141\246y\262\2020\306\343\222\2041\306\227\254 \2141\306\270da\2141\306\245\0\247\270S\353\326\2555!!\301\351\306\343QV\255Z\225\255\252\2615\247{UA$$$\220\232\232\352tc\214\361("\262\313\325t;\304d\2141\306%+c\2141.YAc\214q\311
> \302c\214KV\306c\rkA\210\3104I\221"y\251\216eo\221"\222#"/\210Hp\265y1"\262HD\362Ed\227\210\\355\316\234\306c\352\346\356=\210}\300\303\300'ZHDF\323\201\363\201\240+\360`\265Ef\305@0\230#"\211n\316j\2141\346\334z\204\252.\221\240 @#<buffer *ein: http://127.0.0.1:8888/03_classification.ipynb*>
> 21:34:54:908: [verbose] handle: ( *mm*-929521 ... base64 nil nil nil nil nil) @#<buffer *ein: http://127.0.0.1:8888/03_classification.ipynb*>
> 21:34:54:908: [verbose] type: image/png @#<buffer *ein: http://127.0.0.1:8888/03_classification.ipynb*>
> 21:34:54:908: [verbose] method: nil @#<buffer *ein: http://127.0.0.1:8888/03_classification.ipynb*>
> 21:34:54:908: [warn] ein:cell-append-mime-type: no viewer method found in mailcap @#<buffer *ein: http://127.0.0.1:8888/03_classification.ipynb*>
>

**Comment by dickmao** [comment]
Either google "configuring mailcap on windows" or follow FAQ to inline images.

**Comment by OldIMP** [comment]
Thx for the suggestion. I didn't use EIN for a while and didn't catch up w/ the deactivation of inline images.

Looks like there's no good solution for mailcap on windows w/o the extra installation of Cygwin / Meadow. I guess I'll just turn to browser for figures.
